tests/Equinox.EventStore.Integration/StoreIntegration.fs

```diff
@@ -1,5 +1,6 @@
 ﻿module Equinox.Store.Integration.StoreIntegration
 
+open System.Diagnostics
```

Comment:
:D maybe L56?

Reply:
🤔 I need it open since I start an activity for each test now

Reply:
Also for the SSS and Esdb tests is what I mean

Reply:
yeah really just wanted the sort - all good